### PR TITLE
Adding End to End timing

### PR DESCRIPTION
**Pull request by EricBeach**

### CODE REVIEW

File: src/main/java/com/google/cloud/pontem/Util.java [hunk 1]
```diff
@@ -93,13 +96,39 @@
...
+      String projectId, String instanceId, int numDatabases) {
+    LOG.info("Begin getting list of database names");
+    SpannerOptions options = Util.getSpannerOptionsBuilder().build();
+    Spanner spanner = options.getService();
+
+    DatabaseAdminClient dbAdminClient = spanner.getDatabaseAdminClient();
+
+    List<String> databaseNames = new ArrayList<>();
+
+    try {
```
Comment (ldanielmadariaga) [inline src/main/java/com/google/cloud/pontem/Util.java]: Can you use try with resources and have it close Spanner or it doesn't implement the interface?

Comment (EricBeach) [reply src/main/java/com/google/cloud/pontem/Util.java]: I don't understand your question.

Comment (ldanielmadariaga) [reply src/main/java/com/google/cloud/pontem/Util.java]: If the Spanner interface implements closeable you could do

try (Spanner spanner = options.getService()) {
  Your code
}

While avoiding the finally statement.
See: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Comment (EricBeach) [reply src/main/java/com/google/cloud/pontem/Util.java]: I do not believe Spanner implements the Closable interface.
You can disregard the E2E tests failing. I've run repeatedly locally and I'm trying to figure out what is up with Travis environment.

**Pull request merged**